WcaOnRails/app/views/polls/edit.html.erb

```diff
@@ -54,6 +54,7 @@
       <div class="col-sm-offset-2 col-sm-10">
         <%= f.button :submit, class: "btn-primary" %>
         <% if !@poll.confirmed %>
+          <%= link_to button_tag('Preview', type: "button", class: "btn btn-info"), polls_preview_path %>
```

Comment:
What happens if you make some changes, and then click this button? It looks to me like you'll lose your changes. Does our `are-you-sure` plugin detect this? You know how we added the alert "This poll is open to <%= link_to "voting", polls_vote_path(@poll) %>. Editing question or options is not allowed." to the top of the page? How about something similar like "This poll is not open to voting. You can see a preview *here*. Click confirm when you're ready to enable voting!"